src/mxarray.jl

```diff
@@ -406,13 +406,13 @@ end
 # char arrays and string
 
 function mxarray(s::String)
-    utf16string = transcode(UInt16, s)
+    utf16string = transcode(mxChar, s)
```

Comment:
Right.

1. They are the same so why change it?
2. You are calling julia API defined on `UInt16` and documented as so and why confuse the reader about what you are calling using an matlab type alias?